castero/players/mpvplayer.py

```diff
@@ -83,6 +82,16 @@ def seek(self, direction, amount) -> None:
         if self._player is not None:
             self._player.seek(direction * amount)
 
+    def seek_from_start(self, amount) -> None:
+        """ forward or backward in the media.
+
+        Overrides method from Player; see documentation in that class.
+        """
+        if self._player is not None:
+            # Wait for player to start before seek
+            self._player.wait_until_playing()
+            self._player.seek(amount)
```

Comment:
I think a better way to do this is e.g. `self._player.start = "00:13:14"`. It's documented here, but there might be other similar methods: https://mpv.io/manual/master/#options-start

The way you currently have would actually freeze the client for me, but I didn't look too far into it.